src/service/fileMaker.ts

```diff
@@ -36,6 +28,40 @@ export async function makeMigration(config: Configuration, filename: string): Pr
   }
 
   const timestamp = getTimestampString();
+
+  switch (config.migration.sourceType) {
+    case 'sql':
+      log(`Creating sql migration. ${migrationPath}/${filename}`);
+
+      return makeSqlMigration(filename, migrationPath, timestamp);
```

Comment:
Maybe we can create a generic function `makeMigration` for all 3 types and pass in the extension every time?